### PR TITLE
#185106834 setup empty react project with vite

### PR DESCRIPTION
**Pull request by katros1**
### What does this PR do?

- This PR is for setting up a frontend react project  app 

### How should this be manually tested?

**Installation**
**1. Clone the repository by running** 
```
git clone https://github.com/atlp-rwanda/destructors-ec-fe.git
```
**2. Navigate to the project directory by running **
```
cd destructors-ec-fe
```
```
git checkout ch-setup-reactProject-185106834
```
**3. Install the necessary dependencies by running **
```
npm install
```
**6. Run react  development server by running**
```
npm run dev
```

### Questions:

- none

### CODE REVIEW

**Review by buka1calvin — APPROVED** [review]

**Comment by mjeanbosco19** [comment]
**@katros1** Thank you for doing this as soon as possible. I see you put up a react project using simply Vite which is good, but I don't see a CSS framework to use, like Tailwind CSS?

**Review by mjeanbosco19 — COMMENTED** [review]
@katros1 Thank you for doing this as soon as possible. I see you put up a react project using simply Vite which is good, but I don't see a CSS framework to use, like Tailwind CSS?

**Comment by buka1calvin** [comment]
that's in charge of divine bro

**Comment by katros1** [comment]
> @katros1 Thank you for doing this as soon as possible. I see you put up a react project using simply Vite which is good, but I don't see a CSS framework to use, like Tailwind CSS?

yh I should have worked on that also but there are someone else who is going to take care of that.

**Review by ba-lambert — APPROVED** [review]
You have 2 commits

File: src/components/App.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react'
```
Comment (leandreAlly) [inline src/components/App.jsx:1]: remove app.jsx from components you're changing vite folder structure, and even app.jsx is not components

File: public/vite.svg
Comment (leandreAlly) [inline public/vite.svg:1]: we don't need this

File: src/assets/react.svg
Comment (leandreAlly) [inline src/assets/react.svg:1]: we don't need this

**Review by leandreAlly — CHANGES_REQUESTED** [review]

**Review by Abdulkeza — APPROVED** [review]

**Pull request merged**